lib/src/ast/sass/statement/silent_comment.dart

```diff
@@ -12,11 +13,49 @@ class SilentComment implements Statement {
   /// The text of this comment, including comment characters.
   final String text;
 
+  /// The text of this comment that are marked as a documentation comment
+  /// (leading triple slashes).
+  ///
+  /// Provided with slashes and common leading whitespace removed.
+  String get docComment => _cleanup(text);
+
   final FileSpan span;
 
   SilentComment(this.text, this.span);
 
   T accept<T>(StatementVisitor<T> visitor) => visitor.visitSilentComment(this);
 
+  // Matches '///' followed by some amount of whitespace.
+  final _leadingWhitespace = RegExp('^(\/\/\/[\t ]*)[^\t ]?');
+
+  // Returns a string formed from lines of this comment that begin with '///'
+  // trimmed to remove the slashes and common leading whitespace.
+  String _cleanup(String text) {
+    // Only lines with leading '///'.
+    var lines = text.split('\n').where((line) => line.startsWith('///'));
+    if (lines.isEmpty) return null;
+
+    // Count the common whitespace after '///' on all lines.
```

Comment:
I think we just want to get rid of the first space on each line, not all common whitespace. It'll make the logic much simpler, and it'll allow users to write documentation that's just indented code blocks if they want.

Reply:
Done.